tests/fast-integration/eventing-test/eventing-test.js

```diff
@@ -154,18 +154,21 @@ describe("Eventing tests", function () {
 
   after(async function() {
     // runs once after the last test in this block
-    console.log("Cleaning: Test namespaces should be deleted")
-    await cleanMockTestFixture(mockNamespace, testNamespace, true);
+
+    // Unregister SKR resources from Compass
+    if (isSKR) {
+      debug("Cleaning SKR...")
+      await cleanCompassResourcesSKR(director, appName, scenarioName, skrInfo.compassID);
+    }
 
     // Delete eventing backend secret if it was created by test
     if (eventMeshSecretFilePath !== "") {
```

Comment:
Would check for 'truthy' also work? `if (eventMeshSecretFilePath)` ?

Reply:
updated